### PR TITLE
Fix `pr-branches` and `remove-projects-tab` features

### LINKED ISSUE #1935: console errors
- [ ] On pr page

```
❌ pr-branches
content.js:1865 Error: GraphQL:
Argument 'number' on Field 'NaN' has an invalid value. Expected type 'Int!'.
Argument 'number' on Field 'NaN' has an invalid value. Expected type 'Int!'.
Argument 'number' on Field 'NaN' has an invalid value. Expected type 'Int!'.
Argument 'number' on Field 'NaN' has an invalid value. Expected type 'Int!'.
Argument 'number' on Field 'NaN' has an invalid value. Expected type 'Int!'.
Argument 'number' on Field 'NaN' has an invalid value. Expected type 'Int!'.
Argument 'number' on Field 'NaN' has an invalid value. Expected type 'Int!'.
Argument 'number' on Field 'NaN' has an invalid value. Expected type 'Int!'.
Argument 'number' on Field 'NaN' has an invalid value. Expected type 'Int!'.
Argument 'number' on Field 'NaN' has an invalid value. Expected type 'Int!'.
Argument 'number' on Field 'NaN' has an invalid value. Expected type 'Int!'.
Argument 'number' on Field 'NaN' has an invalid value. Expected type 'Int!'.
Argument 'number' on Field 'NaN' has an invalid value. Expected type 'Int!'.
Argument 'number' on Field 'NaN' has an invalid value. Expected type 'Int!'.
Argument 'number' on Field 'NaN' has an invalid value. Expected type 'Int!'.
Argument 'number' on Field 'NaN' has an invalid value. Expected type 'Int!'.
Argument 'number' on Field 'NaN' has an invalid value. Expected type 'Int!'.
```

- [ ] Almost all pages

```
❌ remove-projects-tab
content.js:1865 TypeError: Cannot read property 'parentElement' of null
    at content.js:2490
    at content.js:478
    at remove_projects_tab_init (content.js:2501)
    at async features_run (content.js:1860)
```

- [x] Doesnt always give the error but it stopped working: https://github.com/sindresorhus/refined-github/issues/1940

```
❌ show-names
content.js:1865 TypeError: Cannot read property 'name' of undefined
    at show_names_init (content.js:4586)
    at async details.init (content.js:1903)
    at async features_run (content.js:1860)

```

### PR DESCRIPTION
**Pull request by HardikModha**
1. `pr-branches`: 
   - An alphanumeric string was being passed to `Number()` causing it to return `NaN` and eventually lead to failure of GraphQL query.
   - https://github.com/sindresorhus/refined-github/issues/1935#issuecomment-484849946

2. `remove-projects-tab`: Selector was renamed from `HeaderMenu` to `Header`.

3. `branch-buttons`: Feature scope was set to `isRepo` so it was breaking on some due to lack of the expected selector. Enabled the features on `isRepoRoot` and `isSingleFile`.

4. `releases-tab`
   - https://github.com/sindresorhus/refined-github/issues/1935#issuecomment-484849946

Fixes #1935
Closes #1935

### CODE REVIEW

File: source/features/pr-branches.tsx [hunk 1]
```diff
@@ -67,13 +67,13 @@ function normalizeBranchInfo(data: BranchInfo): {
...
-function buildQuery(numbers: number[]) {
+function buildQuery(issueIds: string[]) {
 	const {ownerName, repoName} = getOwnerAndRepo();
 
 	return `{
 		repository(owner: "${ownerName}", name: "${repoName}") {
-			${numbers.map(number => `
-				${number}: pullRequest(number: ${String(number).replace('issue_', '')}) {
+			${issueIds.map(id => `
+				${id}: pullRequest(number: ${String(id).replace('issue_', '')}) {
```
Comment (bfred-it) [inline source/features/pr-branches.tsx]: It’s already a string now, so this can be dropped

File: source/features/pr-branches.tsx [hunk 2]
```diff
@@ -67,13 +67,13 @@ function normalizeBranchInfo(data: BranchInfo): {
 	};
 }
 
-function buildQuery(numbers: number[]) {
+function buildQuery(issueIds: string[]) {
```
Comment (bfred-it) [inline source/features/pr-branches.tsx]: No need to rename the variables, while not of type `number` they are still issue “numbers”

File: source/features/pr-branches.tsx (hunk 1 above)
Comment (HardikModha) [reply source/features/pr-branches.tsx]: Duh..Missed it. 😅

File: source/features/pr-branches.tsx (hunk 2 above)
Comment (HardikModha) [reply source/features/pr-branches.tsx]: But IMHO, `issueIds` make more sense and are relevant to the context. They are "issue" ids.

Comment (HardikModha) [reply source/features/pr-branches.tsx]: It's actually `issue_1234` not just `1234` so I changed it to `issueIds`.

**Comment by yakov116** [comment]
@HardikModha want to tackle this last one?

https://github.com/sindresorhus/refined-github/edit/master/.editorconfig
```
❌ branch-buttons
content.js:1880 TypeError: Cannot read property 'textContent' of null
    at getDefaultBranchLink (content.js:3760)
    at async Promise.all (index 0)
    at async branch_buttons_init (content.js:3783)
    at async features_run (content.js:1875)
```

**Comment by HardikModha** [comment]
@yakov116 I'm not seeing this error. The feature is working fine for me.

File: source/features/pr-branches.tsx (hunk 2 above)
Comment (bfred-it) [reply source/features/pr-branches.tsx]: 🤦‍♂️ indeed, I thought they were just `1234`

**Comment by bfred-it** [comment]
> @yakov116 I'm not seeing this error. The feature is working fine for me.

It's there on the linked page

> content.js:1880 TypeError: Cannot read property 'textContent' of null

This is because the feature runs on all `isRepo` pages, it should be reduced to where the required elements appear (without adding more `if`s) like `isRepoTree`, `isSingleFile`, `isCommit`, etc

**Comment by HardikModha** [comment]
Damn...and here I was thinking of why @yakov116 has linked `.editorconfig` here. :sweat_smile:

**Comment by bfred-it** [comment]
Wrong direction, follow this:

> This is because the feature runs on all isRepo pages, it should be reduced to where the required elements appear (without adding more ifs) like isRepoTree, isSingleFile, isCommit, etc

It's not specifically about "Edit File" but it's `isRepo` was too broad

**Comment by HardikModha** [comment]
@bfred-it  So we just add? 🤔  

```

include: [
    features.isRepoTree,
	features.isSingleCommit,
	features.isSingleFile,
],
```

**Comment by bfred-it** [comment]
Yep, and maybe more that I don't remember

**Comment by HardikModha** [comment]
@bfred-it  I went through all the pages and only pages having `.breadcrumb` are getting covered in the two features namely `features.isRepoRoot` and `features.isSingleFile`.

**Comment by bfred-it** [comment]
I think this feature only makes sense when there’s a branch selector, because the feature is an extension of that, but we can probably detect this via page detects

**Comment by bfred-it** [comment]
Thanks @HardikModha 🕊

I dropped the changes related to my latest comment in the other thread as that requires a proper solution in `view-markdown-source`

**Pull request merged**